messaging/src/main/java/org/axonframework/eventhandling/deadletter/jpa/JpaSequencedDeadLetterQueue.java

```diff
@@ -0,0 +1,716 @@
+         * {@link EventMessageDeadLetterJpaConverter}.
+         *
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public JpaSequencedDeadLetterQueue.Builder<T> clearConverters() {
+            this.converters.clear();
+            return this;
+        }
+
+        /**
+         * Adds a {@link DeadLetterJpaConverter} to the configuration, which is used to deserialize items entries from
+         * the database.
+         *
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public JpaSequencedDeadLetterQueue.Builder<T> addConverter(DeadLetterJpaConverter<EventMessage<?>> converter) {
+            this.converters.add(converter);
+            return this;
+        }
+
+        /**
+         * Sets the processing group, which is used for storing and quering which event processor the deadlettered item
+         * belonged to.
+         *
+         * @param processingGroup The processing group of this {@link SequencedDeadLetterQueue}.
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public JpaSequencedDeadLetterQueue.Builder<T> processingGroup(String processingGroup) {
+            this.processingGroup = processingGroup;
+            return this;
+        }
+
+        /**
+         * Sets the claim duration, which is the time a message gets locked when processing and waiting for it to
+         * complete. Other invocations of the {@link #process(Predicate, Function)} method will be unable to process a
+         * sequence while the claim is active. Its default is 30 seconds.
+         * <p>
+         * Claims are automatically released once the item is requeued, the claim time is a backup policy in case of
+         * unforeseen trouble such as down database connections.
+         *
+         * @param claimDuration The longest claim duration allowed.
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public JpaSequencedDeadLetterQueue.Builder<T> claimDuration(Duration claimDuration) {
+            this.claimDuration = claimDuration;
+            return this;
+        }
+
+        /**
+         * Initializes a {@link JpaSequencedDeadLetterQueue} as specified through this Builder.
+         *
+         * @return A {@link JpaSequencedDeadLetterQueue} as specified through this Builder.
+         */
+        public JpaSequencedDeadLetterQueue<T> build() {
+            return new JpaSequencedDeadLetterQueue<>(this);
+        }
+
+        /**
+         * Validate whether the fields contained in this Builder are set accordingly.
+         *
+         * @throws AxonConfigurationException When one field asserts to be incorrect according to the Builder's
+         *                                    specifications.
+         */
+        protected void validate() {
+            if (processingGroup == null) {
```

Comment:
Why aren't the `EntityManagerProvider` and `TransactionManager` in this validation snippet too? 
With a `BuilderUtils#assertNonNull` invocation, for example. 
They're just as important for the workings of this DLQ.